config/initializers/raven.rb

```diff
@@ -0,0 +1,5 @@
+require 'raven'
+
+Raven.configure do |config|
+  config.dsn = ENV['SENTRY_DSN']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.